docs/features/techdocs/using-cloud-storage.md

```diff
@@ -98,6 +101,20 @@ techdocs:
       credentials: ${GOOGLE_APPLICATION_CREDENTIALS}
 ```
 
+Assuming the service account you are using was created in the same project as
```

Comment:
Maybe turning it around, stating when you need it rather than when you don't. Something like: "If the service account used is not in the same project as the bucket, you need to set the `projectId` field."